### PR TITLE
Lcd pins

### PR DESCRIPTION
**Pull request by mass-anis**
added the lcd pins to stm32-pinctrl

### CODE REVIEW

File: scripts/genpinctrl/stm32-pinctrl-config.yaml [hunk 1]
```diff
@@ -234,3 +234,12 @@
 
 - name: USB
   match: "^USB_(?:DM)?(?:DP)?(?:NOE)?$"
+
+- name: LCD_SEG
+  match: "^LCD(?:\\d+)?_SEG\\d+$"
```
Comment (gmarull) [inline scripts/genpinctrl/stm32-pinctrl-config.yaml:239]: please use a single entry, as e.g.

```
^QUADSPI_(?:CLK|NCS|BK1_NCS|BK1_IO[0-3]|BK2_NCS|BK2_IO[0-3])$
```

File: dts/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ URL:
    https://github.com/STMicroelectronics/STM32_open_pin_data
 
 Commit:
-   84963babb708a3465cf4913a83a9d7cb8e5ac13f
+   76cea512bdcafd6be002393400ebd925aec90531
```
Comment (gmarull) [inline dts/README.md:34]: if using another version, first run the generator with the new version and then add LCD support.

**Review by gmarull — CHANGES_REQUESTED** [review]
Please check contribution guidelines to format commit messages accordingly.

File: dts/README.md (hunk 2 above)
Comment (erwango) [inline dts/README.md:34]: Please don't modify SHA1 in current PR.

**Review by erwango — CHANGES_REQUESTED** [review]
Please update commit messages to stick with main repo rules.

File: scripts/genpinctrl/stm32-pinctrl-config.yaml [hunk 3]
```diff
@@ -234,3 +234,6 @@
 
 - name: USB
   match: "^USB_(?:DM)?(?:DP)?(?:NOE)?$"
+
+- name: LCD
+  match: "^LCD\\d*_(?:SEG|COM|VLCD)?\\d*$"
```
Comment (gmarull) [inline scripts/genpinctrl/stm32-pinctrl-config.yaml:239]: Are there any `LCD0_`, `LCD1_`, etc.?

Comment (gmarull) [reply scripts/genpinctrl/stm32-pinctrl-config.yaml:239]: does, e.g. `LCD0_SEGn` exist?

**Comment by erwango** [comment]
Stale, closing

**Pull request closed without merging**